CONTRIBUTING.md

```diff
@@ -54,9 +54,11 @@ Once you have a GitHub login:
 
         $  git clone https://github.com:your-username/exiv2.git
 
-3. Next, create a branch for your PR:
+3. Next, create a branch for your PR from `exiv2_upstream/master` (this is important to avoid draging in old commits):
 
-        $ git checkout -b my-new-feature
+        $ git checkout exiv2_upstream/master
+		$ git checkout -b my-new-feature
```

Comment:
You got two tabs instead of spaces here. Otherwise very nice!

Reply:
Hm, the emacs md-mode preview did not show this but some C-c C-s p magic fixed it.